addons/l10n_ec/data/account_tax_template_vat_data.xml

```diff
@@ -7,40 +7,40 @@
         <!-- IVA 5% -->
         <record id="tax_vat_05_411_goods" model="account.tax.template">
```

Comment:
I don't know if the xml_id must be changed @kenluzuriaga0 @pepetreshere. The base code changed, so I suppose the xml_id changed too.

Reply:
We can do in master